src/documents/APIChangesFrom2_0To3_0.md

```diff
@@ -17,10 +17,17 @@ This section describes major APIs that were in common use. For information about
 
 #### Panel Changes ####
 
+"Links and Buttons" adjuster and enactors collapsed to a single preference, "Enhance Links"
+
 ##### Message Bundle Keys #####
 
 ###### Additions ######
 
+* enhanceInputs.json
```

Comment:
Would be nice to mention the removal of `inputsLarger.json`, `emphasizeLinks.json`, "linksControls.json`.